girder_annotation/girder_large_image_annotation/__init__.py

```diff
@@ -58,6 +59,10 @@ def validateBoolean(doc):
     constants.PluginSettings.LARGE_IMAGE_ANNOTATION_HISTORY: True,
 })
 
+# Access flags
+
+registerAccessFlag('createAnnots', 'Create annotations', 'Allow user to create annotations')
```

Comment:
Can we define this access flag (the `createAnnots`) in `girder/girder_large_image/constants.py` (maybe at the root level of that modules as `ANNOTATION_ACCESS_FLAG = 'large_image_create_annotations'`), and then use the constant instead of the explicit string?

Reply:
See 99ada00